deadlines/src/main/java/com/palantir/deadlines/DeadlinesHttpHeaders.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package com.palantir.deadlines.api;
+package com.palantir.deadlines;
 
 public interface DeadlinesHttpHeaders {
```

Comment:
thoughts on making this package private?

Reply:
i think i'm fine with that. there are some usages outside of this repository but should be easy enough to clean up

Reply:
Are the external uses all in test code?

Reply:
> Are the external uses all in test code?

yes